src/Profiles/Shifting/ANTPLUS_ShiftingPrivateDefines.h

```diff
@@ -0,0 +1,23 @@
+#ifndef ANTPLUS_SHIFTINGPROFILEPRIVATEDEFINES_h
+#define ANTPLUS_SHIFTINGPROFILEPRIVATEDEFINES_h
+
+/* Channel Config */
+#define ANTPLUS_SHIFTING_CHANNELTYPE CHANNEL_TYPE_BIDIRECTIONAL_RECEIVE 
+#define ANTPLUS_SHIFTING_DEVICETYPE 34
+#define ANTPLUS_SHIFTING_CHANNELPERIOD 8192
+// Master channel
+#define ANTPLUS_SHIFTING_MASTER_CHANNELTYPE 0x10 
+#define ANTPLUS_SHIFTING_MASTER_TRANSMISSIONTYPE 5
+#define ANTPLUS_SHIFTING_MASTER_DEVICENUMBER 5
```

Comment:
What is this used for?

Reply:
Due to ANT+ spec, master channel needs these values. See D00001615_ANT+_Device_Profile_-_Shifting_Rev_1.0.pdf section 5.1. Did I misunderstand ?

Reply:
Names changed